special-pages/pages/new-tab/app/favorites/components/PragmaticDND.js

```diff
@@ -0,0 +1,282 @@
+import { h, createContext } from 'preact';
```

Comment:
Everything relating to Pragmatic D+D is placed in this single file. This will make it easier to audit and to swap out if we need to